src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -58,6 +58,13 @@
    :dependencies (schema/pred sequential?)
    :fns-map FnsMap})
 
+(def ProtocolandDependenciesMap
+  "Schema defining the map used to represent the protocol and dependencies
+  of a service."
+  {:fns [clojure.lang.PersistentList]
```

Comment:
this is probably ok, but it seems like it might be overly specific and it'd be nicer if we didn't have to reference a Java class.  I'm guessing that you tried some other things like `(schema/pred seq?)` and they didn't work?